modules/api_docs/test/TestPlan.md

```diff
@@ -0,0 +1,53 @@
+# API Docs module Test Plan
+
+##  Overview
+
+The API docs modules takes the api descriptor of LORIS's modules and creates an 
+actionnable html page. Each endpoint of the selected schema is described and 
+can be tried providing the required parameters. It uses 
+[Swagger-ui](https://github.com/swagger-api/swagger-ui) to create the webpage.
+
+##  Features
+
+1. Presence of LORIS Menu Item 
+
+2. Load schema from various modules.
+
+3. Describe each endpoint's request body and expected responses. 
+
+4. "Try it out" button to execute actions.
+
+**_Note 1:_** Ideally, we should be testing each and every endpoints of all the 
+definitions to make sure that requests and responses definitions matches the 
+actual endpoints behaviour. This is out of scope for this module's tests plan 
+and relies on the modules themselves to ensure that. 
+
+---
+## Permissions
+
+A user requires `API documentation` (`api_docs`) permission to access the 
```

Comment:
Why are the permissions described in the testplan and not the README?

Reply:
I see it in the README
https://github.com/aces/Loris/tree/main/modules/api_docs#permissions